### ISSUE TITLE
Quick Navigation menu bug 

### ISSUE THREAD

**Issue by DrPandemic** [adobe/brackets#10731]
Hi,
I'm running Cinnamon on Arch Linux x64. I just updated my Brackets to "Release 1.2 experimental build 1.2.0-0".
![screenshot from 2015-03-09 21 18 58](https://cloud.githubusercontent.com/assets/3250155/6567691/416ae27c-c6a2-11e4-9ca8-5705a4dc46b9.png)

When I open the Quick Open or Quick Find Definition, the new menu appear over the bar and hides the text box. Also, I'm unable to type to perform a search. The issue persists even if I reload without extensions.

**Comment by prafulVaishnav** [comment]
@DrPandemic  Duplicate of https://github.com/adobe/brackets/issues/9283. As a workaround, typing will correct the position of popup and bar will be displayed. Let me know if it works.

**Comment by DrPandemic** [comment]
I can't type. My keystrokes are ignored when I open the menu.

**Comment by peterflynn** [comment]
@DrPandemic Does the typing problem still happen if you do _Debug > Reload Without Extensions_?  Also, are you able to use other "quick search bar" functions like Ctrl-G (goto line) or Ctrl-T (quick find definition) or are those broken too?

**Comment by DrPandemic** [comment]
Thanks for your time guys. But the bug magically disappeared today after a reboot... My setup didn't change since yesterday, no update, nothing. That's weird, because while I had the bug, I did restart and the bug stayed. Should I close the issue?

**Comment by peterflynn** [comment]
Yep, closing makes sense.  We can always reopen if it comes back & you're able to gather more info.

**Comment by DrPandemic** [comment]
Ok, during some coding, it happend again. I was using ctrl+p, and for no apparent reason, it started to bug. I started a new Brackets instance (without closing the old one) on the same project/folder and in the new instance I didn't had the bug. I restarted the new instance, and it had the bug this time.
The bug is present even if I reload without extensions.
Quick find definition is affected too.
I said that my keystrokes were ignored, that's not completely true. When I close the menu, I can see for a fraction of second that my keystrokes were typed in the text box, but the box was hidden under the menu. Even if the box contains the text, the menu is not updated.
I have the impression that my Brackets settings could be corrupted or something related. What do you think?

**Comment by DrPandemic** [comment]
It could be related (but I really doubt it). I don't have node on my system I have io.js

**Comment by prksingh** [comment]
I have seen this issue but typing something fixes it. You can try clearing Brackets settings data from: C:\Users\<username>\AppData\Roaming\Brackets on Windows -- Don't delete the extensions folder.

If that doesn't work, can you please select *Debug > Show Developer Tools* and check to see if the console contains any errors.

**Comment by DrPandemic** [comment]
You are right, I could clean my settings. But a bug like that is, I think, a signal that maybe the setting parsing/validation could be improve? Currently, my logs or full of : 
```
WebSocket connection to 'ws://localhost:8888/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
```
Maybe a sevice crashed? I have the impression, take this with a grain of salt, that the bug appears when I start working on a file named : profile.ejs.

**Comment by nethip** [comment]
@DrPandemic Could you check to see if this is happening with the Linux released builds? We still have some issues to be fixed in the prerelease build(which has CEF 2171 integrated).

**Comment by DrPandemic** [comment]
The Linux release build doesn't have the problem. Should I close this issue?

**Comment by peterflynn** [comment]
No, I think we should leave it open with a tag indicating it's related to the CEF upgrade, since we wouldn't want to ship that in a Linux release without fixing this first.  @nethip, are you able to repro this on your Linux machine that has the new CEF branch building?

**Comment by nethip** [comment]
@peterflynn I am not able to repro this on my Ubuntu VM.

**Comment by nethip** [comment]
I am using Ubuntu 14.04.1

**Comment by peterflynn** [comment]
Hmm, since Arch Linux isn't officially supported (only Debian & Ubuntu are on the official list) maybe this is an Arch-specific problem?

@DrPandemic Can you try this:

1. Clone the Brackets git repo and checkout the `pflynn/in-browser-file-system branch` branch (and be sure you run `git submodule update --init`)
2. Run a local webserver from the root of the repo
3. Open `localhost:8080/src/index/html` in _Chrome_

Are you unable to enter text in the same places in the UI there?

Another thing to test: go to cefbuilds.com, find the Branch 2171 section, expand Linux 32bit, and find the "CEF 3.2171.1902" build, and click the _"Test App"_ download link.  Run that app, and enter the same URL from step 3 above.  Does the problem happen when viewing the Brackets UI in that link?

**Comment by DrPandemic** [comment]
I didn't know you were planning to have a browser version, I like the idea.

You wrote : ```localhost:8080/src/index/html``` but I used : ```localhost:8080/src/index.html ```

For the first part, I wasn't able to reproduce the bug, but I don't know if it's really representative. The problem was not always present. Normally, I'm able to trigger it when I open files, and in the web version, I can't do much.
For the CEF, I had to run the x64 version, I hope it's ok. For some reason, I'm unable to go on the brackets page. I simply get this error : 
```
Failed to load URL localhost:8080/src/index.html with error (-302).
```
Also, I a previous message in the thread, I said I had : 
```
WebSocket connection to 'ws://localhost:8888/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED
```
Please don't mind it. It is related to an extension.

I hope it was helpful
